data/data-pipeline/data_pipeline/etl/sources/census_acs/etl_imputations.py

```diff
@@ -84,50 +68,58 @@ def calculate_income_measures(
 
     Returns: non-geometry pd.DataFrame
     """
+
+    raw_fields = []
+    imputed_fields = []
+    rename_dict = {}
+    for impute_var in impute_var_named_tup_list:
```

Comment:
Why not just do this in a dict comprehension, like 
```
rename_dict = {impute_var.raw_field_name: impute_var.imputed_field_name for impute_var in impute_var_named_tup_list}
```
? Is there a particular danger that duplicate values in the impute_var_named_tup_list poses? Then you can also eliminate `raw_fields` and `imputed_fields` because they'd be the `keys()` and `values()` of the rename dict.